frontend/apps/reader/modules/readerlink.lua

```diff
@@ -576,56 +628,60 @@ function ReaderLink:onGotoLink(link, neglect_current_location, allow_footnote_po
             })
         end
         return true
-    end
+    elseif self.ui.wallabag then
+        local settings = G_reader_settings:readSetting("external_link_action")
```

Comment:
Forgot to add the menu item to set this setting? Or just undocumented for now?
(`local settings =` plural seems like a bad name for that variable :)

Reply:
I copied this from ReaderStatus. I wouldn't have put the settings in yet if I'd written it from scratch.